Common/UnitDefinitions/VolumeFlow.json

```diff
@@ -450,6 +462,22 @@
           "Abbreviations": [ "af/d" ]
         }
       ]
+    },
+    {
+      "SingularName": "CubicCentimeterPerMinute",
+      "PluralName": "CubicCentimetersPerMinute",
+      "FromUnitToBaseFunc": "x*0.00000001667",
```

Comment:
Counted 7 zeros after the decimal (verified [here](http://convert-to.com/conversion/flow/convert-cm3-per-min-flow-to-m3-per-sec-flow.html))

Reply:
I made the cubic centimeters per minute conversion factor more accurate. The test became correct at a value of **1 m^3/s = 6e7 sm^3/min**